src/circllhist.c

```diff
@@ -1115,8 +1115,10 @@ void
 hist_clear(histogram_t *hist) {
   int i;
   ASSERT_GOOD_HIST(hist);
+  // just to be sure, clear the counts
```

Comment:
C comments here not c++

Reply:
We already have single line `//` comments all over `circllhist.c`. 
Should this be fixed in the whole file ?

Reply:
Sigh. No